src/common/sourceUtils.ts

```diff
@@ -330,3 +330,58 @@ export function escapeRegexSpecialChars(str: string, except?: string): string {
   const r = new RegExp(`[${useRegexChars}]`, 'g');
   return str.replace(r, '\\$&');
 }
+
+export function fileUrlToPath(urlOrPath: string): string {
```

Comment:
We have this as `fileUrlToAbsolutePath` in urlUtils

Reply:
Oh sorry, I missed that! Change to use urlUtils